client/components/account-status/account-status-item.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Flex, FlexBlock, FlexItem } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+
+const AccountStatusItem = ( { label, align, value, children } ) => {
+	return (
+		<Flex
+			direction={ 'row' }
+			align={ align || 'center' }
+			justify={ 'left' }
+			gap={ 3 }
+			className={ 'woocommerce-account-status-item' }
+		>
+			<FlexItem>{ label }</FlexItem>
+			<FlexBlock>{ children || value || null }</FlexBlock>
```

Comment:
same comment as https://github.com/Automattic/woocommerce-payments/pull/1464/files#r637967666

Reply:
> Any update here? @dmvrtx

That's a good point! I changed account-status-settings to re-use component, but decided against extracting it further as it will be removed from settings page soon.

Reply:
> same comment as https://github.com/Automattic/woocommerce-payments/pull/1464/files#r637967666

Here `children` and `value` is something provided by React depending on component usage. I'll leave it as-is for now.